gnd/src/main/java/com/google/android/gnd/ui/home/featuresheet/RecordListViewModel.java

```diff
@@ -57,23 +71,48 @@ public void loadRecordSummaries(Feature feature, Form form) {
         feature.getProject(), feature.getFeatureType().getId(), form.getId(), feature.getId());
   }
 
+  /**
+   * Attempts to fetch a list of records based on the contents of a {@link RecordSummaryRequest}.
+   *
+   * @param request A record summary request. A triple of project, featureId, and formId.
+   * @return A list containing fetched records with forms that satisfy the formId
+   *     provided in the request.
+   */
```

Comment:
We really only need Javadoc on public methods, but ok to keep this if you think it's useful.